src/relax/ir/block_builder.cc

```diff
@@ -600,6 +596,27 @@ class Normalizer : public BlockBuilderImpl, private ExprFunctor<Expr(const Expr&
       UpdateStructInfo(call, inferred_sinfo);
     }
 
+    // In the spirit of joy and child-like whimsy, the conditional
+    // statement below is provided in both C++ and tongue-twister
+    // form.  As very few C++ compilers are fluent in tongue-twister,
+    // only the C++ form is uncommented.
+    //
+    // How much opt could an opt op Op if an opt op could op opt?
+    if (auto opt_op = op->op.as<Op>()) {
+      auto op = opt_op.value();
+      if (apply_f_normalize_ && op_map_normalize_.count(op)) {
```

Comment:
We can use this function https://github.com/apache/tvm/blob/main/include/tvm/ir/op.h#L476

Reply:
Thank you, and updated!  I had checked for a single-parameter `.get`, and an iterator-style `.find`, but hadn't found the two-parameter `.get`.

Updated to use `if (auto func_normalize = op_map_normalize_.get(call->op, nullptr); func_normalize != nullptr)`, here and in `well_formed.cc`.